package.json

```diff
@@ -23,7 +23,7 @@
   },
   "readmeFilename": "README.md",
   "scripts": {
-    "test": "npm run lint && npm run docs && node ./test/all.js",
+    "test": "eslint . && metatests test/",
```

Comment:
Why did you remove the previous changes? This should be
`"test": "npm run lint && npm run docs && metatests test/"`.